client/src/components/navbar-component/navbar.module.css

```diff
@@ -0,0 +1,61 @@
+/* Navbar Styling */
+
+.navbar {
+    /*
```

Comment:
Maybe change the comments since it's describing style from the hack school navbar...